setup.py

```diff
@@ -24,8 +24,9 @@
     "python-dateutil~=2.6",
     "pytz",  # uses calendar versioning
     "scipy>=0.13.3,<2.0",
-    'scikit-learn>=0.18.1,<0.20; python_version < "3.7"',
-    'scikit-learn>=0.19.2,<0.20; python_version >= "3.7"',
+    'scikit-learn>=0.23.1; python_version >= "3.6"',
+    'scikit-learn~=0.19.2, <20.0; python_version < "3.6"',
```

Comment:
Maybe this:
`'scikit-learn>=0.18.1,<0.20; python_version < "3.6"',`

Are we supporting 0.23.1 for 3.6 and above and leaving <3.6 to what is there before?

Reply:
I think 0.18.1 is almost 5 versions behind the previous one. I kept the versioning for python<3.6 to 19.2, because 0.23.1 doesn't support below python 3.7. I don't see any significant changes (in the changelog) that would impact us if we shift from 0.18.1 to 0.19.2.
Is there any additional tests you would want to confirm that it's safe to move to 0.19.2?

Reply:
Hey @vijay120 , let me know what you would like me to do here.